Realm/Tests/PerformanceTests.m

```diff
@@ -591,8 +591,65 @@ - (void)testCommitWriteTransactionWithListNotification {
     }];
 }
 
+- (void)testCommitWriteTransactionWithObjectNotifications {
+    RLMRealm *realm = [self getStringObjects:5];
+    NSMutableArray *tokens = [NSMutableArray new];
+    for (StringObject *so in [StringObject allObjectsInRealm:realm]) {
+        [tokens addObject:[so addNotificationBlock:^(__unused BOOL deleted, __unused NSArray *changes, __unused NSError *error) {
+            CFRunLoopStop(CFRunLoopGetCurrent());
+        }]];
+    }
+
+    // Object notifiers don't have an initial notification, so trigger a
+    // small unmeasured notification we can wait for
+    [realm beginWriteTransaction];
+    [[StringObject allObjectsInRealm:realm].firstObject setStringCol:@"a"];
+    [realm commitWriteTransaction];
+    CFRunLoopRun();
+
+    [self measureMetrics:self.class.defaultPerformanceMetrics automaticallyStartMeasuring:NO forBlock:^{
+        [realm beginWriteTransaction];
+        for (StringObject *so in [StringObject allObjectsInRealm:realm]) {
+            so.stringCol = @"a";
+        }
+        [realm commitWriteTransaction];
+
+        [self startMeasuring];
```

Comment:
What are we measuring exactly on this performance test?, The time committing changes to the 5 objects?

Reply:
The time to produce the change notification, which will stop the run loop once it's delivered.